Lib/blackrenderer/backends/sweepGradient.py

```diff
@@ -0,0 +1,58 @@
+from math import pi, ceil, sin, cos
+from fontTools.misc.vector import Vector
+
+
+def buildSweepGradientPatches(
+    colorLine,
+    center,
+    radius,
+    startAngle,
+    endAngle,
+    useGouraudShading,
+):
+    """Provides colorful patches that mimic a sweep gradient, for use, in
+    particular, in the Cairo and CoreGraphics backends, since these libraries
+    lack the sweep gradient feature."""
+    patches = []
+    # generate a fan of 'triangular' bezier patches, with center 'center' and radius 'radius'
+    degToRad = pi / 180.0
+    if useGouraudShading:
+        maxAngle = pi / 360.0
```

Comment:
So this will give us 720 triangles, right? Perhaps this should be an argument to buildSweepGradientPatches, so we can balance performance with fidelity.

Reply:
Yes, at least 720 in a full disk. At 360 I was still seeing color fringes. Since we are not aiming for performance here, I think this value is reasonable for now.

Reply:
I do intend to use the CG backend in FontGoggles, so performance will become a bit of an issue at that point. But we can fix that later. (https://github.com/justvanrossum/fontgoggles/issues/162)

Reply:
I did add maxAngle as an optional parameter.